service/awsconfig/v7/cloudformation/cloud_formation.go

```diff
@@ -25,9 +26,51 @@ func New(config Config) (*CloudFormation, error) {
 	return c, nil
 }
 
-// TODO
-func (c *CloudFormation) DescribeOutputs(stackName string) ([]*cloudformation.Output, error) {
-	return nil, nil
+func (c *CloudFormation) DescribeOutputsAndStatus(stackName string) ([]*cloudformation.Output, string, error) {
```

Comment:
In the first place I thought I can sort this in a different way but apparently I cannot. So we have to check the outputs and the status at the same time.